contribs/gmf/src/services/objecteditingquery.js

```diff
@@ -220,3 +220,17 @@ gmf.ObjectEditingQuery.GetQueryableLayerNodesResponse;
  * }}
  */
 gmf.ObjectEditingQuery.QueryableLayerInfo;
+
```

Comment:
I tested using `@export` in https://closure-compiler.appspot.com/home#code%3D%252F%252F%2520%253D%253DClosureCompiler%253D%253D%250A%252F%252F%2520%2540compilation_level%2520ADVANCED_OPTIMIZATIONS%250A%252F%252F%2520%2540output_file_name%2520default.js%250A%252F%252F%2520%253D%253D%252FClosureCompiler%253D%253D%250A%250A%252F%252F%2520ADD%2520YOUR%2520CODE%2520HERE%250Agoog.provide(%27toto%27)%253B%250A%250A%252F**%250A%2520*%2520%2540typedef%2520%257B%257B%250A%2520%2520%2520ramonus%253A%2520string%250A%2520%257D%257D%250A%2520*%252F%250Atoto%253B%250A%250A%252F**%250A%2520*%2520%2540type%2520%257Bstring%257D%250A%2520*%2520%2540export%250A%2520*%252F%250Atoto.prototype.ramonus%253B%250A%250A%252F**%250A%2520*%2520%2540param%2520%257Btoto%257D%2520name%250A%2520*%252F%250Afunction%2520hello(name)%2520%257B%250A%2520%2520alert(%27Hello%252C%2520%27%2520%252B%2520name)%253B%250A%257D%250Ahello(generate())%253B%250A%250Afunction%2520generate()%2520%257B%250A%2520%2520return%2520%257B%250A%2520%2520%2520%2520ramonus%253A%2520Math.random()%250A%2520%2520%257D%253B%250A%257D but as you can see it did not prevent the renaming of the attribute to `a`.